### PR TITLE
ingress yaml fix

### PR DESCRIPTION
**Pull request by PixelRobots**
v1beta1 is now not supported so I have updated to use v1

### CODE REVIEW

File: docs/reference-architectures/containers/aks/secure-baseline-aks-content.md [hunk 1]
```diff
@@ -295,8 +295,10 @@ spec:
       paths:
       - path: /
         backend:
-          serviceName: aspnetapp-service
-          servicePort: http
+          service
```
Comment (ckittel) [inline docs/reference-architectures/containers/aks/secure-baseline-aks-content.md:298]:
```suggestion
          service:
```

**Review by ckittel — CHANGES_REQUESTED** [review]
Thanks @PixelRobots!  Just one small typo.

**Review by ckittel — APPROVED** [review]
LGTM

**Comment by ckittel** [comment]
#sign-off

**Comment by PRMerger-2** [comment]
@ckittel: I'm sorry - only the author of this article, @, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by PRMerger8** [comment]
@ckittel: I'm sorry - only the author of this article, @, can sign off on your changes. But we do have an exception process - if you are on the Microsoft content or product team for this product area, you can ask the PR review team to review and merge it by sending mail to the techdocprs alias.

**Comment by PixelRobots** [comment]
> Thanks @PixelRobots! Just one small typo.

Thanks for noticing that. Stupid mistake of mine.

**Pull request merged**